lib/virtual_device.js

```diff
@@ -121,6 +121,27 @@ VirtualDevice.prototype.call = function(/* transition, args, cb */) {
 
 };
 
+VirtualDevice.prototype.available = function(transition) {
+  return !!this._getAction(transition);
+};
+
+VirtualDevice.prototype.transitionsAvailable = function() {
```

Comment:
Yeah it came from zetta-device, I wouldn't be opposed to making them both private.